test/modules/drawer.js

```diff
@@ -37,7 +37,7 @@
             const done = assert.async();
             createViewer();
             assert.ok(viewer.drawer, 'Drawer exists');
-            assert.equal(viewer.drawer.canRotate(), ['webgl','canvas'].includes(drawerType), 'we can rotate if we have canvas');
+            assert.equal(viewer.drawer.canRotate(), ['webgl','webgl2','canvas'].includes(drawerType), 'we can rotate if we have canvas, webgl, or webgl2');
```

Comment:
We should define one place that lists all drawers and tests should access this list from a single place. There are more such tests that run the test item for each drawer type to test them all...